AlfrescoApp/Utils/Categories/UIColor+Custom.h

```diff
@@ -37,5 +37,6 @@
 + (UIColor *)onboardingOffWhiteColor;
 + (UIColor *)siteActionsBackgroundColor;
 + (UIColor *)addTagButtonTintColor;
++ (UIColor *)selectedCollectionViewCellBackground;
```

Comment:
Minor: try to follow the naming convention of existing methods (i.e. end with "Color")